doc/api/fs.md

```diff
@@ -5089,6 +5153,46 @@ characters directly to the `prefix` string. For instance, given a directory
 `prefix` must end with a trailing platform-specific path separator
 (`require('path').sep`).
 
+### `fsPromises.mkstemp(prefix[, options])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `prefix` {string}
+* `options` {string|Object}
+  * `encoding` {string} **Default:** `'utf8'`
+* Returns: {Promise}
+
+Creates and opens a unique empty temporary file, and resolves the `Promise` with
+an object that contains the created file path and a `FileHandle` object. A
+unique file name is generated by appending six random characters to the end of
+the provided `prefix`. Due to platform inconsistencies, avoid trailing `X`
+characters in `prefix`. Some platforms, notably the BSDs, can return more than
+six random characters, and replace trailing `X` characters in `prefix` with
+random characters.
+
+The optional `options` argument can be a string specifying an encoding, or an
+object with an `encoding` property specifying the character encoding to use.
+
+```js
+const fs = require('fs');
+
+async function createRandomFile(path, data) {
+  const result = await fsPromises.mkstemp(path.join(os.tmpdir(), 'foo-'));
+  console.log(result.path);
+  // Prints: /tmp/foo-itXde2 or C:\Users\...\AppData\Local\Temp\foo-itXde2
+  await result.handle.writeFile(data);
+  await result.handle.close();
+}
+createRandomFile('./').catch(console.error);
```

Comment:
This is missing the `data` argument.

Reply:
It might just be better to drop the `writeFile` so that the example is just like the one in `fs`.